doc/user/radial_and_axial_expansion.rst

```diff
@@ -60,27 +57,188 @@ length, :math:`L_0`,
-:eq:`expNewRelative` to find,
+Using Equations :eq:`hotExp` and :eq:`nonRefExp`, we can simplify Equation :eq:`expNewRelative` to find,
 
 .. math::
     \frac{L_h - L_c}{L_c} = \frac{\alpha(T_h) \left(T_h - T_0\right) - \alpha(T_c)\left(T_c - T_0\right)}{1 + \alpha(T_c)\left(T_c - T_0\right)}.
     :label: linearExpansionFactor
 
 Equation :eq:`linearExpansionFactor` is the expression used by ARMI in
-:py:meth:`linearExpansionFactor
-<armi.materials.material.Material.linearExpansionFactor>`.
+:py:meth:`linearExpansionFactor <armi.materials.material.Material.linearExpansionFactor>`.
 
 .. note::
     :py:meth:`linearExpansionPercent
     <armi.materials.material.Material.linearExpansionPercent>` returns
     :math:`\frac{L - L_0}{L_0}` in %.
+
+Given that thermal expansion (or contraction) of solid components must conserve mass throughout the
+system, the density of the component is adjusted as a function of temperature based on the following
+equation, assuming isotropic thermal expansion.
+
+.. math::
+    \rho(T_h) = \frac{\rho(T_0)}{\left(1 + \frac{\Delta L}{L_0}\right)^3} = \frac{\rho(T_0)}{\left(1 + \alpha_m (T_h) (T_h - T_0)\right)^3}
+
+where, :math:`\rho(T_h)` is the component density in :math:`\frac{kg}{m^3}` at the given temperature
+:math:`T_h`, :math:`\rho(T_0)` is the component density in :math:`\frac{kg}{m^3}` at the reference
+temperature :math:`T_0`, and :math:`\alpha(T_h)` is the mean coefficient of thermal expansion at the
+specified temperature :math:`T_h` relative to the material's reference temperature.
+
+An update to mass densities is applied for all solid components given the assumption of isotropic
+thermal expansion.  Here we assume the masses of non-solid components (e.g., fluids or gases) are
+allowed to change within the reactor core model based on changes to solid volume changes. For
+instance, if solids change volume due to temperature changes, there is a change in the amount of
+volume left for fluid components.
+
+Implementation Discussion and Example of Radial and Axial Thermal Expansion
+===========================================================================
+
+This section provides an example thermal expansion calculation for a simple cylindrical component
+from a reference temperature of 20°C to 1000°C with example material properties and dimensions as
+shown in Table :numref:`thermal_exp_comp_properties`.
+
+.. list-table:: Example Component Properties for Thermal Expansion
+   :widths: 50 50
+   :header-rows: 1
+   :name: thermal_exp_comp_properties
+
+   * - Property
+     - Example
+   * - Material
+     - Steel
+   * - Radius
+     - 0.25 cm
+   * - Height
+     - 5.0 cm
+   * - Reference Temperature
+     - 20°C
+   * - Density
+     - 1.0 g/cc
+   * - Mean Coefficient Thermal Expansion
+     - 2 x 10-6 1/°C
+
+Figure :numref:`therm_exp_illustration` illustrates the thermal expansion phenomena in both the
+radial and axial directions.
+
+.. figure:: /.static/axial_expansion_simple.png
```

Comment:
Looking at the PDF and HTML, this is a comically large figure lol. Can we add `:scale: 60` or something to get it to be a better size?

Reply:
try now

Reply:
Still kinda huge imo. And the Figure caption isn't showing 
![image](https://github.com/user-attachments/assets/15468d04-e3fa-4573-b329-4e9315762810)

Reply:
Updated

Reply:
@albeanth Okay, the figure all look fine sized to me now. And the labels are all working.

Tell me what you think.